src/ol/style/expressions.js

```diff
@@ -868,6 +882,17 @@ Operators['atan'] = {
   },
 };
 
+Operators['sqrt'] = {
+  getReturnType: function () {
+    return ValueTypes.NUMBER;
+  },
+  toGlsl: function (context, args) {
+    assertArgsCount(args, 1);
+    assertNumbers(args);
+    return `sqrt(${expressionToGlsl(context, args[0])})`;
```

Comment:
If the value is always static, an alternative is to compute the expression here once and for all instead of passing it to the shader where it will be computed many times.

```suggestion
    return expressionToGlsl(context, Math.sqrt(args[0]))`;
```

Reply:
See #13739 for requirement

Reply:
Thanks for the inputs.